src/TwoBucks/UpdateProfile.java

```diff
@@ -1,75 +1,96 @@
+/**
+ * Update Profile Feature US.24
+ *
+ * The UpdateProfile class is used when the user chooses to update
+ * their old first name, last name, or email to a new one. It will
+ * follow the pipe and filter architecture style because it uses data retrieved
+ * from another portions of the program, edits it, and outputs it in
+ * its original format
+ *
+ * @author Frino Jais
+ * @version 1.0
+ * @since 03-04-2020
+ */
+
 import java.util.*;
 import java.io.*;
 
 public class UpdateProfile {
-
-    public static void readTxtFile(String[] args) throws Exception{
+    /***
+     * This function will read from the text file where the
+     * User information is stored and change the user's choice
+     * of first name, last name, or email address
+     *
+     * @param args
+     * @throws Exception
+     */
+    public static void readFile(String[] args) throws Exception{
+        // creating object for User.txt where data is saved
         File txtFile = new File("User.txt");
+        // reader created to read from User.txt
         BufferedReader reader = new BufferedReader(new FileReader(txtFile));
-        String dataRows[][] = new String[100][5];
-        String rowItems[] = new String[5],row;
+        String dataRows[][] = new String[100][5]; // creating enough rows for 100 users
+        String rowItems[] = new String[5],row; // 5 values per row
         int numOfRows=0,i;
 
-        while((row=reader.readLine()) != null){
-            if(!row.strip().equals("")){
-                rowItems = row.split(",");
-                dataRows[numOfRows++] = rowItems;
+        while((row=reader.readLine()) != null){ // understanding each value separated by comma
+            if(!line.strip().equals("")){ // check for empty string
+                rowItems = line.split(","); // separating values
+                dataRows[numOfRows++] = rowItems; //adding values to an array
             }
         }
 
         Scanner scnr = new Scanner(System.in);
         int option;
-        System.out.println("What would you like to update?");
+        System.out.println("What would you like to update?"); // menu
         System.out.println("");
         System.out.println("1. First Name");
         System.out.println("2. Last Name");
         System.out.println("3. Email");
-        System.out.println("");
-        System.out.println("Option: ");
 
         option = scnr.nextInt();
 
         if (option == 1){
 
-            int rowIndex = 0;
+            int rowIndex = 0; // set the index to 0 to be the position of first name
 
             System.out.println("Please confirm your existing first name: ");
-            String oldFirstName = scnr.next();
+            String oldFirstName = input.next();
             System.out.println("Please enter your new first name: ");
-            String newFirstName = scnr.next();
+            String newFirstName = input.next();
```

Comment:
consider using setFirstName method from the User class

Reply:
Done